Gui/NodeSettingsPanel.cpp

```diff
@@ -88,6 +88,8 @@ NodeSettingsPanel::NodeSettingsPanel(const MultiInstancePanelPtr & multiPanel,
 
 
     QObject::connect( this, SIGNAL(closeChanged(bool)), NodeUi.get(), SLOT(onSettingsPanelClosedChanged(bool)) );
+    QObject::connect( this, SIGNAL(minimized()), NodeUi.get(), SLOT(onSettingsPanelMinimized(bool)) );
+    QObject::connect( this, SIGNAL(maximized()), NodeUi.get(), SLOT(onSettingsPanelMaximized(bool)) );
```

Comment:
We might have an oopsie here (at least in Qt5)

```
QObject::connect: No such slot Natron::NodeGui::onSettingsPanelMinimized(bool) in ../../Gui/NodeSettingsPanel.cpp:91
QObject::connect: No such slot Natron::NodeGui::onSettingsPanelMaximized(bool) in ../../Gui/NodeSettingsPanel.cpp:92
QObject::connect: No such slot Natron::NodeGui::onSettingsPanelMinimized(bool) in ../../Gui/NodeSettingsPanel.cpp:91
QObject::connect: No such slot Natron::NodeGui::onSettingsPanelMaximized(bool) in ../../Gui/NodeSettingsPanel.cpp:92
```

Reply:
Thank you! These lines are useless